xwiki-pro-macros-api/src/main/resources/templates/userlist/userlist.vm

```diff
@@ -0,0 +1,17 @@
+#set ($users = $params.users)
+#set ($users = $users.split(','))
+#set ($discard = $xwiki.ssx.use('XWiki.Macros.UserListMacroStyleSheet'))
+<ul class="xwiki-userlist">
+#foreach ($user in $users)
```

Comment:
We already have a Velocity macro to display a list of users. See https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/macros.vm#L246 .

    @param arg could be either a string, a reference or a list of them

Reply:
Indeed, I had not noticed that one but the list of user properties to be displayed will have to be configurable from the macro so I'm not sure we will be able to reuse this existing `displayUser` macro.